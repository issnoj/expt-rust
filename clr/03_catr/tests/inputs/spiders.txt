Do not worry, spiders,
I keep house
casually.
