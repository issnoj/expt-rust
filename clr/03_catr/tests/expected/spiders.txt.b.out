     1	Do not worry, spiders,
     2	I keep house
     3	casually.
